soc/silabs/silabs_siwx91x/Kconfig.defconfig

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2024 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+rsource "*/Kconfig.defconfig"
```

Comment:
put inside `if SOC_FAMILY_SILABS_SIWX91X`

Reply:
done.

Reply:
not addressed